docs/src/main/asciidoc/hibernate-search-orm-elasticsearch.adoc

```diff
@@ -1015,6 +1015,88 @@ You can enable AWS request signing in Hibernate Search by adding a dedicated ext
 See link:{hibernate-search-orm-elasticsearch-aws-guide}#aws-configuration-reference[the documentation for the Hibernate Search ORM + Elasticsearch AWS extension]
 for more information.
 
+[[management]]
+== Management endpoint
+
+:extension-status: preview
+include::{includes}/extension-status.adoc[]
```

Comment:
I'm not sure it makes sense to do this at the section level; it's not the whole extension we're talking about, but just one feature.

Maybe do something similar to what we did for outbox-polling coordination instead? See https://quarkus.io/guides/hibernate-search-orm-elasticsearch#coordination